aat/src/resources/features/F-018/S-003.td.json

```diff
@@ -0,0 +1,28 @@
+{
+  "title": "should return a negative response when activity is unknown",
+  "_guid_": "S-003",
+  "_extends_": "F-018_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD",
+    "is to add an activity for the case created above",
+    "contains an unknown activity"
+  ],
+
+  "request": {
+
+    "body": {
+      "activity": "open"
+    }
+  },
+    "expectedResponse": {
+      "responseCode": 422,
+      "headers": {
+        "_extends_": "Common_422_Response_Headers"
```

Comment:
Same as the missing activity

Reply:
raised a ticket https://tools.hmcts.net/jira/browse/RDM-8986